mesop/component_helpers/style.py

```diff
@@ -73,12 +73,26 @@ class BorderSide:
                                 a string with a unit (e.g., '2em'), or None for no width.
       color: The color of the border, represented as a string. This can be any valid CSS color value,
                           or None for no color.
-      style: The style of the border, which can be 'none' for no border, 'solid' for a solid line.
+      style: The style of the border.
```

Comment:
Can you update the comment to also include a link where the types of border styles are. Usually we use developer.mozilla.org links: https://developer.mozilla.org/en-US/docs/Web/CSS/border-style

Reply:
Hey, so should i just add that link to the comment?
I've committed the file to just see if all the checks pass, I'll make these changes next.